src/Middleware/CasProxyTicketSessionMiddleware.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\Middleware;
+
+use Http\Promise\Promise;
+use Phpro\SoapClient\Middleware\Middleware;
+use Phpro\SoapClient\Middleware\MiddlewareInterface;
+use Psr\Http\Message\RequestInterface;
+use Symfony\Component\HttpFoundation\Session\Session;
+
+class CasProxyTicketSessionMiddleware extends Middleware implements MiddlewareInterface
```

Comment:
we need to add documentation here on the scope of the class.

Reply:
Done.

Reply:
No that does not explain what's the scope of the class! :smile:

Reply:
Done better.